e2e-tests/integration/eventEditingTest.js

```diff
@@ -130,9 +130,9 @@ context('Event editing:', () => {
 
     // add a new datetime
     // advance calendar one month and select the 10th
-    cy.get('#cal-container .v-date-picker-header > button:last-child').click()
+    cy.get('#cal-container .flatpickr-next-month').click()
     cy.wait(1000) // wait for calendar animation to complete
```

Comment:
We can deal with this later, but the `wait` might not be necessary with Flatpickr -- it was just there because we couldn't figure out a way to wait on the Vuetify one's animation.

Reply:
yeah wondered about that, but didn't want to change more than I needed

Reply:
removed 26c88da4454a1de00277cade533e92664483984d